tests/SDKTest.php

```diff
@@ -44,6 +44,19 @@ class SDKTest extends TestCase
             ],
         ],
 
+        'java' => [
+            'class' => 'Appwrite\SDK\Language\Java',
+            'build' => [
+                'mkdir -p tests/sdks/java/src/test/java/io/appwrite/services',
+                'cp tests/languages/java/ServiceTest.java tests/sdks/java/src/test/java/io/appwrite/services/ServiceTest.java',
+                'docker run --rm -v $(pwd):/app -w /app/tests/sdks/dart --env PUB_CACHE=vendor maven:3.6-jdk-11-slim mvn clean install',
+            ],
+            'envs' => [
+                'java-11' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/java --env PUB_CACHE=vendor maven:3.6-jdk-11-slim mvn clean install',
+                'java-14' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/java --env PUB_CACHE=vendor maven:3.6-jdk-14-slim mvn clean install',
+            ],
+        ],
+
         'typescript' => [
             'class' => 'Appwrite\SDK\Language\TypeScript',
             'build' => [
```

Comment:
Need to whitelist 'java' on line 139 so tests will actually run

Reply:
Done.